akka-docs/rst/java/http/client-side/connection-level.rst

```diff
@@ -53,6 +53,10 @@ The connection can also be closed by the server.
 An application can actively trigger the closing of the connection by completing the request stream. In this case the
 underlying TCP connection will be closed when the last pending response has been received.
 
+Connection will also be closed if response entity has been cancelled (e.g. by attaching it to ``Sink.cancelled()``)
```

Comment:
"The connection will also be closed if the response entity is cancelled" IMHO sounds more right